### PR TITLE
Translation: Library Carpentry OpenRefine Episode 1

### PR DESCRIPTION
**Pull request by naganeayu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,141 @@
...
+小さな貢献をしたい場合は、[クイックスタートガイド](docs/quickstart.md)を読んでみてください。
+
+資料を翻訳する前に、[翻訳者のガイドライン](docs/TranslatorGuidelines.md)と、[行動規則](docs/rules.md)を、読ん
+でください
+
+また、私たちの[Slackワークスペース](http://carpentries-ja.slack.com)に参加してください。
+[#swcarpentryのSlackチャンネル](https://r-wakalang.herokuapp.com/)もあります。
+ここは、翻訳を進める上で発生する疑問に関して質問するのに最適の場所です。
+
+ツイッターで[swcarpentry-ja](https://twitter.com/swcarpentry_ja)も更新の発表をやっています。ぜひフォローし
```
Comment (masamiy) [inline README.md:14]:
```suggestion
ツイッター[swcarpentry-ja](https://twitter.com/swcarpentry_ja)もやっています。ぜひフォローし
```

File: README.md [hunk 2]
```diff
@@ -1,19 +1,141 @@
...
+ソフトウェアカーペントリーのメインのウェブサイトが他の言語と一緒にホストされ互換性があり、英語のレッスンの新しいリリースに合わせて日本語のレッスンも最新に維持
+するように計画しています。
+
+私達は、ソフトウェアカーペントリーのオリジナルのレッスンを改定するのではなく、レッスを翻訳（そして最新の状態に維持）をしています。
+
+もしあなたが、レッスン自体の問題に気づいたら、英語のレッスンに対して、issueをたてたり、プルリクエストを送ってください。
+
+## gitについて
+
+ここでは、あなたが、GitとGitHubについて知っているという仮定をしています。
```
Comment (masamiy) [inline README.md:31]:
```suggestion
ここでは、あなたが、Git と GitHub について知っていると仮定ます。
```

File: README.md [hunk 3]
```diff
@@ -1,19 +1,141 @@
...
+
+リポジトリは、[ソフトウェアカーペントリー](https://software-carpentry.org/)のレッスンを英語から他の言語(現在日本語に取り組
+んでいます)へ翻訳するのを容易にするために必要なファイルやツールをホストするためにあります。また、英語とスペイン語のレッスンの多言語化をしてマージするためでもあります。
+
+ソフトウェアカーペントリーのメインのウェブサイトが他の言語と一緒にホストされ互換性があり、英語のレッスンの新しいリリースに合わせて日本語のレッスンも最新に維持
+するように計画しています。
+
+私達は、ソフトウェアカーペントリーのオリジナルのレッスンを改定するのではなく、レッスを翻訳（そして最新の状態に維持）をしています。
+
+もしあなたが、レッスン自体の問題に気づいたら、英語のレッスンに対して、issueをたてたり、プルリクエストを送ってください。
```
Comment (masamiy) [inline README.md:27]:
```suggestion
もしあなたが、レッスン自体の問題に気づいたら、英語のレッスンに対して、issue をたてたり、プルリクエストを送ってください。
```

File: README.md [hunk 4]
```diff
@@ -1,19 +1,141 @@
...
+
+## gitについて
+
+ここでは、あなたが、GitとGitHubについて知っているという仮定をしています。
+
+もし、gitをインストールするのに手助けが必要ならば、[gitのインストールガイド](docs/git.md)を読んでみてください
```
Comment (masamiy) [inline README.md:33]:
```suggestion
もし、gitをインストールするのにサポートが必要ならば、[gitのインストールガイド](docs/git.md)を読んでみてください
```

File: README.md [hunk 5]
```diff
@@ -1,19 +1,141 @@
...
+
+各レッスンのテキストを直接翻訳するのではなく、翻訳には、[PO ファイル](https://www.gnu.org/software/gettext/manu
+al/html_node/PO-Files.html)を使います。
+
+POファイルは、複数のエントリで構成されるテキスト形式のファイルです。各エントリーは、元のテキストと、その翻訳の短い部分が含まれています。オリジナルの
+レッスンごとに、１つのPOファイルがあります。例えば、
+[Software Carpentry 'git novice' lesson](https://github.com/swcarpentry/git-nov
+ice)は、複数の markdown形式のドキュメントで構成され、日本語の翻訳に関しては１つの `git-novice.ja.po` というPOファイルがあります。これにより、手
+動ではなく、
+オリジナルのテキストの追跡を行うことができ、オリジナルのレッスンが更新されたときに、変更され翻訳が必要とされる部分を正確に把握することができます。
```
Comment (masamiy) [inline README.md:45]:
```suggestion
オリジナルテキストの追跡を行うことができ、オリジナルのレッスンが更新されたときに、変更され翻訳が必要とされる部分を正確に把握することができます。
```

File: README.md [hunk 6]
```diff
@@ -1,19 +1,141 @@
...
+git checkout ja
+git remote add swc-ja git@github.com:swcarpentry-ja/i18n.git
+git pull swc-ja ja
+```
+
+PRがレビューをパスするまでステップ4と5を繰り返します
+
+いくつかのメモ:
+
+POファイルを編集しても、翻訳されたWebサイトができあがるわけではありません。[メンテナーと管理者ガイド](docs/admin.md)で述べられているよう
```
Comment (masamiy) [inline README.md:108]:
```suggestion
PO ファイルを編集しても、翻訳された Web サイトができあがるわけではありません。[メンテナーと管理者ガイド](docs/admin.md)で述べられているよう
```

File: README.md [hunk 7]
```diff
@@ -1,19 +1,141 @@
...
+[#swcarpentryのSlackチャンネル](https://r-wakalang.herokuapp.com/)もあります。
+ここは、翻訳を進める上で発生する疑問に関して質問するのに最適の場所です。
+
+ツイッターで[swcarpentry-ja](https://twitter.com/swcarpentry_ja)も更新の発表をやっています。ぜひフォローし
+てください。
+
+## 目的
+
+リポジトリは、[ソフトウェアカーペントリー](https://software-carpentry.org/)のレッスンを英語から他の言語(現在日本語に取り組
+んでいます)へ翻訳するのを容易にするために必要なファイルやツールをホストするためにあります。また、英語とスペイン語のレッスンの多言語化をしてマージするためでもあります。
```
Comment (masamiy) [inline README.md:20]: I am not 100% sure of the English version of the sentence here, but:
```suggestion
んでいます)へ翻訳するのを容易にするために必要なファイルやツールをホストするためのものです。また、英語やスペイン語など他言語のレッスンをマージするためでもあります。
```

File: README.md [hunk 8]
```diff
@@ -1,19 +1,141 @@
...
+また、私たちの[Slackワークスペース](http://carpentries-ja.slack.com)に参加してください。
+[#swcarpentryのSlackチャンネル](https://r-wakalang.herokuapp.com/)もあります。
+ここは、翻訳を進める上で発生する疑問に関して質問するのに最適の場所です。
+
+ツイッターで[swcarpentry-ja](https://twitter.com/swcarpentry_ja)も更新の発表をやっています。ぜひフォローし
+てください。
+
+## 目的
+
+リポジトリは、[ソフトウェアカーペントリー](https://software-carpentry.org/)のレッスンを英語から他の言語(現在日本語に取り組
```
Comment (masamiy) [inline README.md:19]: Do we have a plan to translate lessons other than Japanese?

リポジトリは、[ソフトウェアカーペントリー](https://software-carpentry.org/)のレッスンを英語から他の言語(**現在日本語に取り組...** sounds like we will work on a different language in the future. 
```

File: README.md [hunk 9]
```diff
@@ -1,19 +1,141 @@
...
+んでいます)へ翻訳するのを容易にするために必要なファイルやツールをホストするためにあります。また、英語とスペイン語のレッスンの多言語化をしてマージするためでもあります。
+
+ソフトウェアカーペントリーのメインのウェブサイトが他の言語と一緒にホストされ互換性があり、英語のレッスンの新しいリリースに合わせて日本語のレッスンも最新に維持
```
Comment (masamiy) [inline README.md:22]:
```suggestion
私たちは、日本語のレッスンが互換性を持って他の言語のレッスンと共にソフトウェアカーペントリーのメインのウェブサイトから提供され、英語のレッスンのアップデートに伴い日本語のレッスンもアップデート出来るようにすることを計画しています。
```

File: README.md [hunk 10]
```diff
@@ -1,19 +1,141 @@
...
+するように計画しています。
+
+私達は、ソフトウェアカーペントリーのオリジナルのレッスンを改定するのではなく、レッスを翻訳（そして最新の状態に維持）をしています。
```
Comment (masamiy) [inline README.md:25]:
```suggestion
私達は、ソフトウェアカーペントリーのオリジナルのレッスンを変更するのではなく、レッスを日本語に翻訳（そして最新の状態を維持）する作業をしています。
```

File: README.md [hunk 11]
```diff
@@ -1,19 +1,141 @@
...
+
+## gitについて
+
+ここでは、あなたが、GitとGitHubについて知っているという仮定をしています。
+
+もし、gitをインストールするのに手助けが必要ならば、[gitのインストールガイド](docs/git.md)を読んでみてください
+
+## About PO files
+
+各レッスンのテキストを直接翻訳するのではなく、翻訳には、[PO ファイル](https://www.gnu.org/software/gettext/manu
```
Comment (masamiy) [inline README.md:37]:
```suggestion
翻訳作業を行う際は、各レッスンのテキストを直接翻訳するのではなく、[PO ファイル](https://www.gnu.org/software/gettext/manu
```

File: README.md [hunk 12]
```diff
@@ -1,19 +1,141 @@
...
+al/html_node/PO-Files.html)を使います。
+
+POファイルは、複数のエントリで構成されるテキスト形式のファイルです。各エントリーは、元のテキストと、その翻訳の短い部分が含まれています。オリジナルの
+レッスンごとに、１つのPOファイルがあります。例えば、
+[Software Carpentry 'git novice' lesson](https://github.com/swcarpentry/git-nov
+ice)は、複数の markdown形式のドキュメントで構成され、日本語の翻訳に関しては１つの `git-novice.ja.po` というPOファイルがあります。これにより、手
```
Comment (masamiy) [inline README.md:43]:
```suggestion
ice)は、複数の markdown形式のドキュメントで構成され、日本語の翻訳に関しては１つの `git-novice.ja.po` という PO ファイルがあります。これにより、
```

File: README.md [hunk 13]
```diff
@@ -1,19 +1,141 @@
...
+動ではなく、
```
Comment (masamiy) [inline README.md:44]:
```suggestion
容易に、
```

File: README.md [hunk 14]
```diff
@@ -1,19 +1,141 @@
...
+ます。
+
+## 初めてレッスンをインポートするとき
+
+レッスンはサブモジュールとしてインポートされます。この作業はレッスンにつき１度だけ行われ、ほとんどの翻訳者はこれを行う必要はありません。
+もし、あなたが新しいレッスンをインポートしたいときには、[インポートのガイド](docs/importing.md)を参照してください。
+
+## 既存のレッスンの翻訳へ貢献する
+
+**これは、翻訳者からの手助けが最も必要なタスクです!**
```
Comment (masamiy) [inline README.md:58]:
```suggestion
**ヘルプが最も必要なタスクがここです!**
```

File: README.md [hunk 15]
```diff
@@ -1,19 +1,141 @@
...
+
+これは、[インポートのガイド](docs/importing.md)で説明されているように、`swcarpentry-ja/i18n`がサブモジュールとして既
+に追加されていることを
+前提としています。あなたは、そのレッスンの翻訳に貢献したいとします。
```
Comment (masamiy) [inline README.md:62]:
```suggestion
前提としています。あなたが、そのレッスンの翻訳に貢献したいとします。
```

File: README.md [hunk 16]
```diff
@@ -1,19 +1,141 @@
...
+
+1. 個人のGitHubアカウントで、このリポジトリの"フォーク"を作成します。
```
Comment (masamiy) [inline README.md:64]:
```suggestion
1. 個人の GitHub アカウントで、このリポジトリの"フォーク"を作成します。
```

File: README.md [hunk 17]
```diff
@@ -1,19 +1,141 @@
...
+  (`https://github.com/swcarpentry-ja/i18n`の右上の"フォーク(fork)"をクリックします。
+  corner of the `https://github.com/swcarpentry-ja/i18n` webpage)
+
+2. 個人のアカウント(GitHubUserなど)で、このリポジトリをクローンします。これは、翻訳ファイルのバージョン管理するためのローカルコピーです
```
Comment (masamiy) [inline README.md:68]:
```suggestion
2. 個人のアカウント(GitHubUser など)で、このリポジトリをクローンします。これは、翻訳ファイルのバージョン管理するためのローカルコピーです。
```

File: README.md [hunk 18]
```diff
@@ -1,19 +1,141 @@
...
+
+```bash
+cd directory
+git clone git@github.com:GitHubUser/i18n.git
+cd i18n
+```
+
+3. サブモジュールを使えるようにする.
```
Comment (masamiy) [inline README.md:76]:
```suggestion
3. サブモジュールを使えるようにします。
```

File: README.md [hunk 19]
```diff
@@ -1,19 +1,141 @@
...
+
+```bash
+git submodule init
+git submodule update
+```
+
+このリポジトリはすでに、レッスンの翻訳ファイルが含まれています。翻訳に貢献したいときには、 `po` ディレクトリの中の `<レッスン名>.<言語>.po`
```
Comment (masamiy) [inline README.md:83]:
```suggestion
このリポジトリはすでに、レッスンの翻訳ファイルが含まれています。貢献は、 `po` ディレクトリの中の `<レッスン名>.<言語>.po` 
```

File: README.md [hunk 20]
```diff
@@ -1,19 +1,141 @@
...
+3. サブモジュールを使えるようにする.
+
+```bash
+git submodule init
+git submodule update
+```
+
+このリポジトリはすでに、レッスンの翻訳ファイルが含まれています。翻訳に貢献したいときには、 `po` ディレクトリの中の `<レッスン名>.<言語>.po` 
+、
+例えば `git-novice.ja.po` に貢献することができます。
```
Comment (masamiy) [inline README.md:85]:
```suggestion
例えば `git-novice.ja.po` から出来ます。
```

File: README.md [hunk 21]
```diff
@@ -1,19 +1,141 @@
...
+
+```bash
+cd po
+ls git-novice.ja.po
+```
+
+4. POファイルを編集します。[ガイドラインに従って](docs/rules.md), 頻繁に変更をコミットし、十分だとおもったらpull request
```
Comment (masamiy) [inline README.md:92]:
```suggestion
4. POファイルを編集します。[ガイドラインに従って](docs/rules.md)、頻繁に変更をコミットし、作業が完了したら pull request 
```

File: README.md [hunk 22]
```diff
@@ -1,19 +1,141 @@
...
+、
+例えば `git-novice.ja.po` に貢献することができます。
+
+```bash
+cd po
+ls git-novice.ja.po
+```
+
+4. POファイルを編集します。[ガイドラインに従って](docs/rules.md), 頻繁に変更をコミットし、十分だとおもったらpull request 
+を投稿してください。
```
Comment (masamiy) [inline README.md:93]:
```suggestion
を送ってください。
```

File: README.md [hunk 23]
```diff
@@ -1,19 +1,141 @@
...
+
+5. 正確を期すために、あなたのPRについて、レビューが行われます。レビューがパスするまで、編集を続けます。そのようなときは、まずはじめに、
```
Comment (masamiy) [inline README.md:95]:
```suggestion
5. 正確を期すために、あなたの PR はレビュアーがレビューします。レビューがパスするまで、編集を続けます。そのようなときは、まずはじめに、
```

File: README.md [hunk 24]
```diff
@@ -1,19 +1,141 @@
...
+組織のリポジトリに対する変更を pull してください。
```
Comment (masamiy) [inline README.md:96]:
```suggestion
swcarpentry-ja/i18n.git リポジトリに対する変更を pull してください。
```

File: README.md [hunk 25]
```diff
@@ -1,19 +1,141 @@
...
+
+```bash
+git checkout ja
+git remote add swc-ja git@github.com:swcarpentry-ja/i18n.git
+git pull swc-ja ja
+```
+
+PRがレビューをパスするまでステップ4と5を繰り返します
+
+いくつかのメモ:
```
Comment (masamiy) [inline README.md:106]:
```suggestion
注意事項:
```

File: README.md [hunk 26]
```diff
@@ -1,19 +1,141 @@
...
+
+POファイルを編集しても、翻訳されたWebサイトができあがるわけではありません。[メンテナーと管理者ガイド](docs/admin.md)で述べられているよう
+に、
+これはメンテナーに任されています。
+
+POファイルを編集したあと、翻訳された Markdown ファイルを見たいときには、 `bash po4gitbook/compile.sh` を実行します。
+これは、
+あなたの変更を含めた翻訳されたバージョンを生成します。これは、`locale/<lang>/<lesson>`, 例,
```
Comment (masamiy) [inline README.md:114]:
```suggestion
あなたの変更を含めた翻訳バージョンを生成します。これは、`locale/<lang>/<lesson>`, 例,
```

File: README.md [hunk 27]
```diff
@@ -1,19 +1,141 @@
...
+`locale/ja/git-novice`で見つけることができます。
+
+## メインの英語のレッスンが新しくリリースされたら、レッスンを更新するのに貢献する
```
Comment (masamiy) [inline README.md:117]:
```suggestion
## 翻訳済レッスンの英語バージョンがアップデートされた場合の貢献
```

File: README.md [hunk 28]
```diff
@@ -1,19 +1,141 @@
...
+
+レッスンの(完全な)翻訳が存在しても、メインの英語のレッスンが新しくリリースされることがあります。
+更新されたバージョンの英語のレッスンは、現在の翻訳されたものにマージされる必要があります。
+
+[レッスン更新時のガイド](docs/updating.md)を見てください。
```
Comment (masamiy) [inline README.md:122]:
```suggestion
[レッスン更新時のガイド](docs/updating.md)を参照してください。
```

File: README.md [hunk 29]
```diff
@@ -1,19 +1,141 @@
...
+
+## 翻訳のためのリソース
+
+レッスンを編集するときには[翻訳者のためのガイドライン](docs/TranslatorGuidelines.md)に従ってください。
+
+レッスン間で用語が一貫して使われていることを確認するために
```
Comment (masamiy) [inline README.md:128]:
```suggestion
レッスンでは一貫した用語が使用されているようにします。 
```

File: README.md [hunk 30]
```diff
@@ -1,19 +1,141 @@
...
+
+[レッスン更新時のガイド](docs/updating.md)を見てください。
+
+## 翻訳のためのリソース
+
+レッスンを編集するときには[翻訳者のためのガイドライン](docs/TranslatorGuidelines.md)に従ってください。
+
+レッスン間で用語が一貫して使われていることを確認するために 
+[技術用語のリスト](https://github.com/swcarpentry-ja/i18n/wiki/Glossary-for-technical-terms)
+ があります。必要に応じてこれを参照したり更新したりしてください。
```
Comment (masamiy) [inline README.md:130]:
```suggestion
必要に応じて[技術用語のリスト](https://github.com/swcarpentry-ja/i18n/wiki/Glossary-for-technical-terms)を参照したり更新したりしてください。
```

File: README.md [hunk 31]
```diff
@@ -1,19 +1,141 @@
...
+更新されたバージョンの英語のレッスンは、現在の翻訳されたものにマージされる必要があります。
+
+[レッスン更新時のガイド](docs/updating.md)を見てください。
+
+## 翻訳のためのリソース
+
+レッスンを編集するときには[翻訳者のためのガイドライン](docs/TranslatorGuidelines.md)に従ってください。
+
+レッスン間で用語が一貫して使われていることを確認するために 
+[技術用語のリスト](https://github.com/swcarpentry-ja/i18n/wiki/Glossary-for-technical-terms)
```
Comment (masamiy) [inline README.md:129]: Suggest to delete.

File: README.md [hunk 32]
```diff
@@ -1,19 +1,141 @@
...
+ があります。必要に応じてこれを参照したり更新したりしてください。
+
+一貫性を担保するために、日本語に翻訳されない概念などの標準化された取り扱いについては、 [カルチャーノート](docs/CultureNotes.md) を参
```
Comment (masamiy) [inline README.md:132]:
```suggestion
一貫性を保持するために、日本語に翻訳されない概念などの標準化された取り扱いについては、 [カルチャーノート](docs/CultureNotes.md) を参
```

File: README.md [hunk 33]
```diff
@@ -1,19 +1,141 @@
...
+照してください。
+
+進捗と、ゴールを記録するために[変更履歴](docs/ChangeLog.md)を持っています。
```
Comment (masamiy) [inline README.md:135]:
```suggestion
進捗と、達成度を記録するために[変更履歴](docs/ChangeLog.md)があります。
```
FYI, I could not access to the ChangeLog.md[https://github.com/swcarpentry-ja/i18n/blob/ja/docs/ChangeLog.md] from the original doc. Is it just me?

File: README.md [hunk 34]
```diff
@@ -1,19 +1,141 @@
...
+
+## メンテナーと管理者のガイド
+
+[メンテナーと管理者のガイド](docs/admin.md) を見てください
```
Comment (masamiy) [inline README.md:139]:
```suggestion
[メンテナーと管理者のガイド](docs/admin.md) を参照してください。
```

File: README.md [hunk 35]
```diff
@@ -1,19 +1,141 @@
...
+## [English README](README_en.md)
+
+小さな貢献をしたい場合は、[クイックスタートガイド](docs/quickstart.md)を読んでみてください。
+
+資料を翻訳する前に、[翻訳者のガイドライン](docs/TranslatorGuidelines.md)と、[行動規則](docs/rules.md)を、読ん
+でください
+
+また、私たちの[Slackワークスペース](http://carpentries-ja.slack.com)に参加してください。
+[#swcarpentryのSlackチャンネル](https://r-wakalang.herokuapp.com/)もあります。
+ここは、翻訳を進める上で発生する疑問に関して質問するのに最適の場所です。
```
Comment (masamiy) [inline README.md:12]:
```suggestion
翻訳作業中に生じた疑問などは、ここから質問してください。
```

**Review by masamiy — CHANGES_REQUESTED** [review]
Thank you for your work!
I think I have reviewed at stuff I should look. If there are specific sections/sentences you would like me to have a look, please do let me know.

**Comment by naganeayu** [comment]
Wrong branch & PR

**Pull request closed without merging**